taskcluster/node-package-cpu.yml

```diff
@@ -13,6 +13,7 @@ build:
   scripts:
     build: "taskcluster/node-build.sh"
     package: "taskcluster/node-package.sh"
+  workerType: "${docker.decisionTask}"
```

Comment:
Looks like `decisionTask` is not the best name for this worker type if it's used for caching and packaging tasks as well?

Reply:
changed to smallTask